### ISSUE TITLE
Win10Support

### ISSUE THREAD

**Issue by zzani** [quasar/QuasarRAT#696]
Hello,

thank you very much for the tool. I wanted to test it with two Win10 in the lab and could not start it on the "victim" OS. Namely  it says that this app can't be run on that pc. Per default .net is installed but probably higher version. If I try to install .net 4.0 i get an error saying that it is already installed.
Can you help what else needs to be done in order to start it?

Thanks

**Comment by MaxXor** [comment]
.NET Framework 4.5 is pre-installed on Windows 10 so something else is blocking it.

**Comment by zzani** [comment]
Can it be that it is because I am using trial win version so something is not activated. Antivirus is disabled (real time) - Do you know what else can be?

**Comment by MaxXor** [comment]
What message is shown when u start it?

**Comment by zzani** [comment]
it is shown generic  "This app can’t run on your PC. To find the version for your pc check with your publisher" message. I disabled UAC with slider only not through GPO, I enabled real admin account and I tried to use run as admin. Also it was written that some SmartScreen is possible as a cause so I disabled that one as well. I restarted the vm to be sure that change was done and now I am getting something else - Operation did not complete successfully because the file contains a virus or potentially unwanted software. It seems that somethign from above did the trick. But still long way from the result. I will check further. Nice for you as well to know that there are some built it def features
